apps/server/src/modules/database/mongo-console/mongo-console.module.ts

```diff
@@ -0,0 +1,20 @@
+import { DynamicModule, Module } from '@nestjs/common';
+import { MongoConsoleService } from './mongo-console.service';
+import { MongoConsoleController } from './mongo-console.controller';
+import { MongoConsoleOptions } from './mongo-console-options.interface';
+
+@Module({})
+export class MongoConsoleModule {
+	static forRoot(options?: MongoConsoleOptions): DynamicModule {
+		if (options != null && options.publish === true) {
```

Comment:
just asking: shouldn't be `options.publish === true` be sufficient here?

Reply:
i will use `options?.publish === true`